src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/permission/PermissionEvaluatorService.java

```diff
@@ -65,43 +67,52 @@ private boolean evaluateAccess(Map<String, CamundaVariable> variables,
                                    List<PermissionTypes> permissionsRequired) {
         boolean hasAccess;
 
+        log.debug("Evaluating access for {}", roleAssignment);
```

Comment:
I think it would be good to turn Debug mode ON in AAT, so that we can see these logs.

Reply:
yes this would be amazing :)

Reply:
If it is not on, we should then to trouble shoot.